private/rules/coursier.bzl

```diff
@@ -963,6 +965,12 @@ def rewrite_files_attribute_if_necessary(repository_ctx, dep_tree):
 
     return dep_tree
 
+def is_dep(jar_path, deps):
+    for dep in reversed(deps):
```

Comment:
why reversed?

Reply:
Good catch. It's a leftover from an optimization I tried. I will remove it.